mathics/builtin/drawing/image.py

```diff
@@ -2083,7 +2083,7 @@ def get_sort_key(self, pattern_sort=False):
         if pattern_sort:
             return super(Image, self).get_sort_key(True)
         else:
-            return hash(self)
+            return [1, 3, SymbolImage, tuple(), 2, len(self.pixels), hash(self)]
```

Comment:
It would be nice to give some sort of human description of why this makes sense.